subsys/net/lib/openthread/platform/radio.c

```diff
@@ -1210,3 +1228,8 @@ otError otPlatRadioConfigureEnhAckProbing(otInstance *aInstance, otLinkMetrics a
 }
 
 #endif /* CONFIG_OPENTHREAD_LINK_METRICS_SUBJECT */
+
+uint16_t otPlatTimeGetXtalAccuracy()
```

Comment:
Notice `otPlatRadioGetCslAccuracy` already exists and uses some Kconfig option as well. Would be nice to avoid duplication if possible.

Reply:
It seems like both options aim to give the same clock property, so it's probably a good idea in general, but I think that `IEEE802154_DELAY_TRX_ACC` would be rather confusing here.

Reply:
Agree. I'd be open to remove `IEEE802154_DELAY_TRX_ACC` in favor of something more generalist.